website/docs/setup/configuration.md

```diff
@@ -127,6 +168,24 @@ directory within the container.
 </TabItem>
 </Tabs>
 
+The config file can also be used to provide other configuration information like Azure and AWS credentials as mentioned below:
```

Comment:
Given the additions above, shouldn't this be removed? It's outside of a tab group.

Reply:
It should have been inside docker's tab group. Moved it there.